### ISSUE TITLE
Missing new client_details fields in mysql schema

### ISSUE THREAD

**Issue by juazugas** [mitreid-connect/OpenID-Connect-Java-Spring-Server#1112]
Hi! 

First, thanks for all the work. We are developing our SSO with Mitreid OpenID Connect. We use Mysql as storage engine. We've detected two few new fields in the table client_details, but they aren't defined in the mysql schema.

The fields are `client_details`. `software_statement` and `client_details`.`code_challenge_method`.  I leave you the definitions.

/openid-connect-server-webapp/src/main/resources/db/tables/mysql_database_tables.sql
...

    software_statement VARCHAR(4096),

    code_challenge_method VARCHAR(256),
...

Also happens the same for PostgreSQL.

Thanks again, 
Juan Zuriaga

**Comment by jricher** [comment]
Correct, the schemas haven't been sync'd yet. Keep in mind that the database schema could change in the development version on `master`, but that it will be stable in the release version on `1.2.x`.

**Comment by juazugas** [comment]
OK, Thanks. I'll be watching.
We are currently in the development phase. Probably we'll continue using the `master` branch until the next release.
